<launch>
    <include file="$(find nlink_parser)/launch/linktrack_5.launch" />
    <param name="use_sim_time" type="bool" value= "false" />
    <node pkg="range_localization" type="range_node" name="playback" output="screen" >
        <rosparam command="load" file="$(find range_localization)/config/uav.yaml" />
        <rosparam command="load" file="$(find range_localization)/config/anchor_position_nooploop.yaml" />
        <param name="vehicle_name"  value="tuav" />
        <param name="imu_topic" value="/tuav/dji_osdk_ros/imu" />
        <param name="range_topic"  value="/tuav_1/nlink_linktrack_nodeframe3" />
        <param name="pressure_topic"  value="/tuav/dji_osdk_ros/height_above_takeoff" />
        <rosparam param="init_position" >[0.0, 0.0, 0.0]</rosparam>
    </node>

    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find range_localization)/rviz_cfg/uwb_loc.rviz" />
    </group>
    <!-- <node pkg="rosbag" type="record" name="suav_2_bag_record" args="/suav_2/uwb/data /suav_2/imu/data /suav_2/filter/odom /suav_2/dji_osdk_ros/height_above_takeoff /suav_2/dji_osdk_ros/vo_position /suav_2/dji_osdk_ros/imu -o /home/$(env USER)/suav_2.bag"/> -->
</launch>
